### DISCUSSION TITLE
SB possible on PinePhone?

### DISCUSSION THREAD

**Discussion by githubques** [markqvist/Reticulum#186] [Sideband]
Looking up LORA's and came across a Linux Phone (think it's been mentioned on here before) that has a LORA add on:

https://pine64.com/product/pinephone-pinephone-pro-pindio-lora-add-on-case/

In the description it mentions a programmer named "JF" whose designed some mesh software for it.

Seems like it would be a good fit for SB/Reticulum since you're always looking for python programmers and apparently that's what a PinePhone (since it's Linux based) relies on.

Because the idea of a non-iPhone/non-Android smart phone is so innovative, it makes it a target for getting kicked off cell phone carriers for whatever reason.

SB could be the back up communication system making sure it still works no matter what. This could be motivation enough to get some Pine programmers to collaborate with you on a SB PinePhone port helping lift some of the coding weight.

**Comment by markqvist** [comment]
Good ideas! As far as I can tell, there's nothing stopping this from working perfectly, other than it simply not being implemented yet.

If I had a couple of PinePhones, I'm pretty sure I could get it up and running quite easily.

Maybe I should reach out to them and hear if they're willing to donate a few devices for having support added.

**Reply by githubques** [reply]
It wouldn't hurt to post an offer to help develop something in exchange for a few donated devices. 

They've launched a new Pro PinePhone so there's likely people sitting on their original ones willing to donate if they know how much SB/Reticulum can improve their user experience.

Their forum: https://forum.pine64.org/forumdisplay.php?fid=121 has a post about I2P end to end encryption:
https://forum.pine64.org/showthread.php?tid=17832

The author may be a good member to message for feedback on a collaboration/how to reach the right people.

"I am flat-out busy"

On my end, python coding just isn't in my wheelhouse otherwise I'd try to contribute more.

**Comment by The-One-Law** [comment]
### I have a pinephone   
 ...and also read with interest some other comments in the PinePhone community
 regarding implementing solutions Pinephone was designed for
 and that Reticulum seems better positioned to provide.

I am flat-out busy - IRL with my other work,
 but if anyone tries a Pinephone Reticulum implementation please let me know
 and I will try to help in whatever manner I can manage.

Pinephone simply begs for a Reticulum implementation,
 for too many reasons to clarify here.
As mentioned above:
` "Because the idea of a non-iPhone/non-Android smart phone is so innovative, it makes it a target for getting kicked off cell phone carriers for whatever reason."`
